deploy/podman/pod-persistent.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: assisted-installer
+  name: assisted-installer
+spec:
+  containers:
+  - args:
+    - run-postgresql
+    image: quay.io/centos7/postgresql-12-centos7:latest
+    name: db
+    envFrom:
+    - configMapRef:
+        name: config
+    volumeMounts:
+      - mountPath: /var/lib/pgsql
+        name: pg-data
+  - image: quay.io/edge-infrastructure/assisted-installer-ui:latest
+    name: ui
+    ports:
+    - hostPort: 8080
+    envFrom:
+    - configMapRef:
+        name: config
+  - image: quay.io/edge-infrastructure/assisted-image-service:latest
+    name: image-service
+    ports:
+    - hostPort: 8888
+    envFrom:
+    - configMapRef:
+        name: config
+  - image: quay.io/edge-infrastructure/assisted-service:latest
+    name: service
+    ports:
+    - hostPort: 8090
+    envFrom:
+    - configMapRef:
+        name: config
+    volumeMounts:
```

Comment:
Not sure that this one is needed. 
What are you trying to get from this one? 
All the clusters are stored in the DB. 
Unless you are talking about kubeconfig that i'm not sure if it's stored in the `/data`.

Reply:
OK it does contain the object handler so it will store the logs, kubeconfig etc https://github.com/openshift/assisted-service/blob/7ea3e5c083d0c1395804c037683970f4187d456a/cmd/main.go#L144 
You can ignore this comment.